frontend/packages/dev-console/integration-tests/features/topology/topology-layout-save.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Topology Layout should be saved
+    As a user, working in the topology, the graphical layout should be remembered for each project
+
+
+    Background:
+        Given user is at the topolgy page
+        And user has selected namespace "aut-topology-layout-save"
+        And user has created workload "hello-openshift"
+        And user is at the topology page
+
+
+    @manual
+    Scenario: Topology Graph View persistance
+        Given user has selected Graph View
+        When user goes to add page
+        And user comes back to topology page
+        Then user will see topology view unchanged
+
+    
+    @manual
+    Scenario: Topology List View persistance
+        Given user has selected List View
+        When user logged out from cluster
+        And user logged in to cluster
+        Then user will see topology view unchanged
+
+    
+    @manual
+    Scenario: Topology layout1 persistance
+        Given user has selected layout1
```

Comment:
Ux defines layouts in capital i.e. Layout 1

Reply:
Updated